sdk/src/wallet/storage/adapter/wasm.rs

```diff
@@ -10,35 +12,82 @@ pub const STORAGE_ID: &str = "Wasm";
 
 /// Wasm storage adapter using the browser local storage
 #[derive(Debug)]
-pub struct WasmAdapter(LocalStorage);
+pub struct WasmAdapter {
+    /// field used just to make it impossible to create a [`WasmAdapter`] without using the `new` function.
+    _ignored: (),
+}
+
+const PROBE_KEY: &str = "iota-sdk-availability-test";
+const PROBE_VALUE: &str = "local storage is available!";
```

Comment:
do we need to keep these as consts or we can put them inside the new() function where they are used

Reply:
Moved them into the `new` function 💯